components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2147,6 +2200,10 @@ where
         if self.fsm.peer.unsafe_recovery_state.is_some() {
             self.check_unsafe_recovery_state();
         }
+        // TODO: combine recovery state and flashback state as a wait apply queue.
+        if self.fsm.peer.flashback_state.is_some() {
+            self.fsm.peer.maybe_finish_flashback_wait_apply();
```

Comment:
check it for `CheckApplyingSnapStatus::Success` as well

Reply:
Do u mean that in /raftstore/src/store/peer.rs#2405?

Reply:
I don't know the exact line number

Reply:
I've written a comment in the specific location below, PTAL.

Reply:
I've written a comment in the specific location below, PTAL.